api-schema/src/main/java/org/stellar/anchor/api/exception/RpcException.java

```diff
@@ -0,0 +1,22 @@
+package org.stellar.anchor.api.exception;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class RpcException extends AnchorException {
+  private final int errorCode;
+  private Object additionalData;
+
+  public RpcException(int errorCode, String message) {
```

Comment:
Should it has RpcErrorCode as parameter instead of int?

Reply:
Changed `int errorCode` to `RpcErrorCode errorCode`